shared/api/auth.go

```diff
@@ -6,6 +6,9 @@ const (
 
 	// AuthenticationMethodOIDC is a token based authentication method.
 	AuthenticationMethodOIDC = "oidc"
+
+	// AuthenticationMethodDevLxd is the authentication method for interacting with the devlxd API..
+	AuthenticationMethodDevLxd = "devlxd"
```

Comment:
This is an internal authentication method, so it should be in `lxd/auth/types.go`